### ISSUE TITLE
Cell Probability

### ISSUE THREAD

**Issue by faraz89** [broadinstitute/CellBender#106]
Thanks for the tool.

**I am not sure how I feel about the determination of which barcodes contain cells. Is it normal for it be so cleanly assigned?**

<img width="280" alt="Screenshot 2021-07-14 at 09 24 12" src="https://user-images.githubusercontent.com/24622959/125589379-5ef83a39-485b-475f-9918-3238adb20d7b.png">

**In addition, I want to clarify the on the "--total-droplets-included" number based on the UMI decay curve below. Do you think 25000 is a reasonable threshold given the UMI curve below?**

<img width="451" alt="Screenshot 2021-07-14 at 09 27 17" src="https://user-images.githubusercontent.com/24622959/125590027-cb547886-093c-4ff3-be80-7d75b01b7191.png">

This the code run:
cellbender remove-background --input /filtered_matrix/ --output DOX_NEG_cellblender.h5 --expected-cells 5000 --total-droplets-included 25000

**I am curious to know when I run the above code, why do I get " 0 empty droplets". Please see below**

<img width="848" alt="Screenshot 2021-07-14 at 09 44 49" src="https://user-images.githubusercontent.com/24622959/125592372-e2662986-877d-4204-b611-856e5797cd78.png">

**Comment by sjfleming** [comment]
Hi @faraz89 ,

I agree that this is not performing as intended.  For some reason, the algorithm is excluding most of your dataset, and it's not seeing those empty droplets.  My guess is that your `expected-cells` parameter is fooling the algorithm a little bit.  It believes that a reasonable prior for counts in empty droplets is 7330, so it thinks that the "empty droplet plateau" is in the wrong place.  This is a really clean looking dataset!

I would suggest trying the following:
```
cellbender remove-background --input /filtered_matrix/ --output DOX_NEG_cellblender.h5 --expected-cells 13000 --total-droplets-included 25000
```

since it looks to me like there are more like 13000 cells in the dataset.  Ideally, the algorithm should be less sensitive to this `expected-cells` parameter.  I am working on trying to remedy this.

**Comment by faraz89** [comment]
Hi @sjfleming,

Many thanks for the response. I ran the code as your suggested but despite the modification I still get "0" empty droplets. Please see below.

<img width="851" alt="Screenshot 2021-07-15 at 09 23 10" src="https://user-images.githubusercontent.com/24622959/125754888-54ec89fa-3919-4b8a-9e41-8e778089d1a8.png">

**Comment by sjfleming** [comment]
Hi @faraz89 , hmm this is puzzling.  I definitely want the code to work on a sample like this!

Can you try adding `--low-count-threshold 5` to the command ?

**Comment by sjfleming** [comment]
This kind of thing should be resolved in v0.3.0

Closed by #238
